libs/shared/src/lib/components/widgets/common/graphql-variables-mapping/graphql-variables-mapping.component.ts

```diff
@@ -55,6 +62,10 @@ export class GraphqlVariablesMappingComponent implements OnChanges {
   /** size style of editor */
   public style: any = {};
 
+  ngOnInit() {
+    this.control.setValidators(validJson());
```

Comment:
```suggestion
    this.control.setValidators(validJson());
    this.control.updateValueAndValidity();
```
https://angular.io/api/forms/AbstractControl#setValidators
Also set validators would replace any other validator set in the control, maybe would be better to use the `addValidators()` method?

KR, Unai